venues/ICLR.cc/2019/Conference/python/review-assignment-stage.py

```diff
@@ -0,0 +1,61 @@
+'''
+Reviewer Assignment Stage:
+- Bidding Task / interface is closed.
+- Initial paper-reviewer and paper-areachair matches are computed.
+- Program chairs browse match using OpenReview Matching System and make hand-adjustments as needed.
+- Paper-reviewer and paper-areachair assignments are finalized and deployed.
+
+'''
+
+import argparse
+import openreview
+import iclr19
+import invitations
+import notes
+import time
+import json
+
+import matcher
+
+def clear(client, invitation):
+    for note in openreview.tools.iterget_notes(client, invitation = invitation):
+        client.delete_note(note)
```

Comment:
Does delete the note from the db?

Reply:
I'm not sure - from the client library, it looks like it's sending a "delete" request to the backend. does that delete it from the db or set a ddate? it may delete it from the db.

would that be a problem?

Reply:
here's the code: https://github.com/iesl/openreview-py/blob/master/openreview/openreview.py#L493

Reply:
It removes the notes from the database, I think it is better than setting a ddate. Because metadata notes are heavy and we don't want to keep them. 

I wonder if for some reason you need to keep them.